c/blake3_dispatch.c

```diff
@@ -76,22 +77,32 @@ enum cpu_feature {
 #if !defined(BLAKE3_TESTING)
 static /* Allow the variable to be controlled manually for testing */
 #endif
-    enum cpu_feature g_cpu_features = UNDEFINED;
+#if defined(_MSC_VER)
+LONG
+#else
+_Atomic int
```

Comment:
MSVC doesn't support `_Atomic` yet when building C code (https://github.com/microsoft/STL/blob/main/stl/inc/stdatomic.h#L15), so the Win32 interlocked APIs have to be used.